packages/web/src/store/application/ui/withdraw-usdc/sagas.ts

```diff
@@ -177,12 +186,207 @@ function* createDestinationTokenAccount({
   )
 }
 
-/**
- * Handles all logic for withdrawing USDC to a given destination. Expects amount in cents.
- */
-function* doWithdrawUSDC({
-  payload: { amount, currentBalance, destinationAddress }
-}: ReturnType<typeof beginWithdrawUSDC>) {
+function* doWithdrawUSDCCoinflow({
+  amount,
+  currentBalance
+}: Pick<
+  ReturnType<typeof beginWithdrawUSDC>['payload'],
+  'amount' | 'currentBalance'
+>) {
+  const { track, make } = yield* getContext('analytics')
+  yield* put(beginCoinflowWithdrawal())
+
+  const analyticsFields: WithdrawUSDCTransferEventFields = {
+    // TODO
```

Comment:
not sure if you intended to leave this todo

Reply:
When I first changed it, I thought we might eventually want to put something different here. But I'm not sure what that would be, so I can remove it for now.